setup.py

```diff
@@ -83,5 +83,5 @@
     package_dir={"": "src"},
     include_package_data=True,
     install_requires=install_requires,
-    scripts=['src/health/azure/run_tensorboard.py']
+    scripts=['src/health/azure/run_tensorboard.py', 'src/health/azure/download_aml_run.py']
```

Comment:
Do we have docs for the cmd line tools? It would be good to add a link from main page

Reply:
Have added to the docs